packages/ampd-sdk/src/grpc/client.rs

```diff
@@ -156,6 +188,34 @@ impl Client for GrpcClient {
 
         Ok(broadcast_response.into())
     }
+
+    async fn contract_state(
+        &mut self,
+        contract: nonempty::String,
+        query: nonempty::Vec<u8>,
+    ) -> Result<Vec<u8>, Error> {
```

Comment:
```suggestion
    ) -> Result<serde_json::Value, Error> {
```
The bytes returns are supposed to be valid JSON.